app.js

```diff
@@ -153,5 +167,3 @@ getForm.addEventListener('submit', (e) => {
   e.preventDefault();
   checkLowerCase(getEmail);
 });
```

Comment:
- [ ] **You have done a great job preserving data on the local storage :medal_sports:**  However, I think it would be better to make the following improvement to complete all the project's requirements 
    - Kindly, after filling out the form, and not submitting it, the entered values should be saved in the browser. you may try to change the used ```EventLister``` from ```' submit'```  to ```'change'```  :+1: :+1: 
       _follow this [link](https://www.javascripttutorial.net/javascript-dom/javascript-change-event/) for more info on eventListener_ :joy: 

   - It would be better also to update and save data to the local storage when the user changes the content of any input field in the form :+1: :+1: